ProteGO/Common/DataModel/GetStatus/GetStatusRequest.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+struct GetStatusRequest: Encodable {
+
+    let userId: String
+
+    let platform: String
+
+    let osVersion: String
+
+    let deviceType: String
+
+    let appVersion: String
+
+    let apiVersion: String
+
+    let lang: String
+
+    enum CodingKeys: String, CodingKey {
+        case userId = "user_id"
+        case platform
+        case osVersion = "os_version"
+        case deviceType = "device_type"
+        case appVersion = "app_version"
+        case apiVersion = "api_version"
```

Comment:
To się zmieniło, wcześniej backend przyjmował te parametry z kluczami camel case. Zamienisz od razu klucze na snake case w `RegisterDeviceRequest` i `ConfirmRegistrationRequest`?

Reply:
@jerzyKleszcz @Axadiw https://developer.apple.com/documentation/foundation/jsondecoder/2949119-keydecodingstrategy
.convertFromSnakeCase ?

Reply:
👍 
ustawiłem to na wszystkich requestach w GcpEndpoint.swift

przy okazji usunąłem to ręczne specyfikowanie kluczy w snake_case dla innych rodzajów requestów